icu4j/maven-build/local-maven-repo/com/ibm/icu/icudata/73.1-SNAPSHOT/icudata-73.1-SNAPSHOT.pom

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.ibm.icu</groupId>
+  <artifactId>icudata</artifactId>
+  <version>73.1-SNAPSHOT</version>
```

Comment:
Will we need to rename and modify these files for every release candidate, release, and post-release phase?
Or is this a temporary situation?

Reply:
That said, this approach is fine for now, but I don't think it's good long term.